extensions/amp-story/1.0/amp-story-share-menu.js

```diff
@@ -48,9 +50,9 @@ const getTemplate = (element) => {
   return htmlFor(element)`
     <div class="i-amphtml-story-share-menu i-amphtml-story-system-reset" aria-hidden="true" role="alert">
       <div class="i-amphtml-story-share-menu-container">
-        <span class="i-amphtml-story-share-menu-close-button" role="button">
+        <button class="i-amphtml-story-share-menu-close-button" aria-label="">
```

Comment:
Why do we have an empty aria-label here?

Reply:
A placeholder aiming to help readability. 
I think it can be helpful if a template gives an overview of the attributes it needs.

Reply:
If you strongly feel it's confusing it can be removed.

Reply:
Oh got it, I don't think I've seen this pattern anywhere else. In general I'd say that attributes that are set in JS don't need to be in the template unless they have a default value, so no need to keep the empty aria-label.

If you want, setting the default `aria-label="Close"` value here and then on the JS do

```js
if (localizationService) {
  button.setAttribute("aria-label", localizationService.getLocalizedString(...));
}
```
it also makes sense, up to you.

Reply:
Agreed it makes sense to put the default value in the template.